src/ir/wrap.jl

```diff
@@ -58,7 +58,11 @@ function IRCode(ir::IR)
   sps = VERSION > v"1.2-" ? [] : Core.svec()
 
   @static if VERSION > v"1.6-"
-    stmtinfo = Any[nothing for _ in 1:length(length(stmts))]
+    @static if isdefined(Core.Compiler, :CallInfo)
+      stmtinfo = Core.Compiler.CallInfo[Core.Compiler.NoCallInfo() for _ in 1:length(length(stmts))]
```

Comment:
Length of length?

Reply:
`length(::Number)` always returns `1`, so I guess this `stmtinfo` actually has been invalid. I will fix it up and see if the CI runs successfully.

Reply:
Okay, CI is successful. This should be ready.